x-pack/solutions/security/plugins/security_solution_serverless/common/pli/pli_config.ts

```diff
@@ -22,8 +22,9 @@ export const PLI_PRODUCT_FEATURES: PliProductFeatures = {
       ProductFeatureKey.configurations,
       ProductFeatureKey.externalDetections,
     ],
-    essentials: [ProductFeatureKey.attackDiscovery, ProductFeatureKey.assistant],
-    complete: [ProductFeatureKey.attackDiscovery, ProductFeatureKey.assistant],
+    // neither of these tiers are available in ai_soc product line
+    essentials: [],
+    complete: [],
```

Comment:
Actually, the `ProductLine.aiSoc` add-on will be available for _essentials_ (not sure _complete_) eventually, right?

Reply:
Maybe, but from the UX and discussions so far it looks like ai4Soc does not have those tiers at all.

Reply:
I thought the plan was to, eventually, be able to add the `ai4Soc` add-on to the _essentials_ tier, to add AI features to it. Has the plan changed?

Reply:
Maybe you are confusing `ai4Soc` add-on with the `searchAiLake` tier?

Reply:
@semd you're right 👍 `ai_soc` will be available in `essentials/complete` in the future.

This comment sums up the current and the next phases: https://github.com/elastic/security-team/issues/11781#issuecomment-2701518004